src/autoscaler/Makefile

```diff
@@ -133,10 +133,10 @@ check: fmt lint build test
 .PHONY: generate-fakes
 test: generate-fakes
 	@echo "Running tests"
-	APP_AUTOSCALER_TEST_RUN='true' go run github.com/onsi/ginkgo/v2/ginkgo -p ${GINKGO_OPTS} ${TEST} --skip-package='integration'
```

Comment:
This has two problems:
- It bypasses the devbox.json by running some ginkgo from somewhere else.
- It does not work anyway because `${TEST}` must come after every option.